File: apps/pos/native-windows/DesktopKitchenPOS/Services/BrandingService.cs
using DesktopKitchenPOS.Models;
using DesktopKitchenPOS.Networking.Endpoints;

namespace DesktopKitchenPOS.Services;

public class BrandingService
{
    public async Task<TenantBranding> GetBrandingAsync()
    {
        return await DesktopKitchenPOS.App.Api.RequestAsync<TenantBranding>(
            HttpMethod.Get, BrandingEndpoints.Branding);
    }
}

File: apps/pos/native-android/app/src/main/java/com/desktopkitchen/pos/services/BrandingService.kt
package com.desktopkitchen.pos.services

import com.desktopkitchen.pos.models.TenantBranding
import com.desktopkitchen.pos.networking.api.BrandingApi
import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class BrandingService @Inject constructor(
    private val brandingApi: BrandingApi
) {
    suspend fun getBranding(): TenantBranding = brandingApi.getBranding()
}
